### DISCUSSION TITLE
How to compute "focal_x_baseline"

### DISCUSSION THREAD

**Discussion by sandeepdutta** [stella-cv/stella_vslam#613] [Q&A]
Hello All,
I am using a Realsense camera , with fx = 424.1945 & baseline (i.e. distance between the two cameras) =  49.21 mm.
What should be the value of focal_x_baseline ? Do I need to convert the baseline from mm to meters ? or just multiply
fx*baseline ?
Thank you in advance. Best Regards
Sandeep

**Comment by Skwangles** [comment]
Yes, fx*baseline if the Realsense is a stereo camera

Docs describe it as [https://stella-cv.readthedocs.io/en/latest/parameters.html#camera](https://stella-cv.readthedocs.io/en/latest/parameters.html#camera)
```
focal_x_baseline

For stereo cameras, it is the value of the baseline between the left and right cameras multiplied by the focal length fx. For RGBD cameras, if the measurement method is stereo, set it based on its baseline. If the measurement method is other than that, set the appropriate value based on the relationship between depth accuracy and baseline.
```
